books.php

```diff
@@ -15,6 +15,18 @@ public static function getInstance()
         }
         return self::$instance;
     }
+    public function auth($login,$password)
+    {
+        $loginPasswordCheck = "SELECT login,password FROM users";

```

Comment:
You can directly check if there is proper user from the beginning without get all users.
1. select id from users where login='' and password='';
2. if length(returned_rows) > 0 then logged in (return true)
Please use prepared statement:
https://websitebeaver.com/php-pdo-prepared-statements-to-prevent-sql-injection#named-parameters